src/articles/Article.js

```diff
@@ -0,0 +1,52 @@
+class Article {
+  constructor(title, content, writer) {
+    this._title = title;
+    this._content = content;
+    this._writer = writer;
+    this._createdAt = new Date();
+    this._likeCount = 0;
+  }
+
+  set title(value) {
+    if (value.length === 0) {
+      throw new Error('제목이 비어있습니다.');
+    }
+    this._title = value;
+  }
```

Comment:
코드를 읽어보면, 필수 값이라고 생각되는 것들에 대해 length로 검증하시는 것 같아요.
필수 값에 대해 벨리데이션한다는 아이디어는 훌륭합니다 👍

현재는 빈 문자열은 잘 걸러내고, null/undefined는 TypeError로 걸리네요.
다만 숫자나 boolean 같은 타입은 통과하게 되는데, 각 파라미터들에 다른 타입이 들어와도 된다고 한다면, 올바른 코드가 되겠고 아니라면 타입 가드를 조금 더 단단하게 구성하는 것도 좋아요.

```suggestion
  set title(value) {
    if (typeof value !== 'string' || value.trim().length === 0) {
      throw new Error('제목은 공백이 아닌 문자열이어야 합니다.');
    }
    this._title = value;
  }
```